gorums.proto

```diff
@@ -14,6 +15,17 @@ extend google.protobuf.MethodOptions {
   bool qc_future = 50004;
   bool qf_with_req = 50005;
   bool per_node_arg = 50006;
+  bool strict_ordering = 50007;
 
   string custom_return_type = 51000;
 }
+
+service Gorums {
```

Comment:
On second thought, I'm not sure we want to expose this RPC and `Message` here, since they are internal to Gorums; should not be used by others. I think I might have suggested this, but you know... Seems like these would fit well in the `strictordering.proto`, or am I missing something?

Reply:
It's "internal" to gorums, but imported by the generated code, so I don't think it's possible to put it in an "internal" package.

Reply:
The service has now been moved to `dev`